### PR TITLE
Fixes 1478, remove rash from Liberia symptoms

### PR DESCRIPTION
**Pull request by readevalprint**
https://sprint.ly/product/22465/item/1478

### CODE REVIEW

File: json/lists.json [hunk 1]
```diff
@@ -74,4 +73,4 @@
       "other_symptoms"
     ],
   "case": []
-}]
\ No newline at end of file
+}]
```
Comment (rmehner) [inline json/lists.json]: Finally somebody with sane editor settings! :)

Comment (readevalprint) [inline json/lists.json]: :D

**Comment by rmehner** [comment]
Thank you, please release that as version so we can use it in Sense. Ping me if you need any help.

**Comment by danse** [comment]
Luckily i have heard of this in the stand-up. This needs to be done in Case Management as well. Also, how this affects existing data? Did you check that this symptom is not already present in production data?

**Comment by danse** [comment]
I created the story at https://sprint.ly/product/22465/item/1506 to track related tasks

**Comment by readevalprint** [comment]
@danse good point!

**Pull request merged**